docs/source/conf.py

```diff
@@ -97,7 +97,6 @@
 
 # Additional templates that should be rendered to pages, maps page names to
 # template names.
-html_additional_pages = {'index': 'index.html'}
```

Comment:
Hack to get `index.html` not good, break all the cross link and force hacks in all the places.